types/knuddels-userapp-backend-api/index.d.ts

```diff
@@ -0,0 +1,2588 @@
+// Type definitions for non-npm package Knuddels UserApps API 1.20210326150104
+// Project: https://developer.knuddels.de
+// Definitions by: Knuddels GmbH & Co. KG <https://github.com/Knuddels>
```

Comment:
The `Definitions by:` list controls who gets pinged for reviews on future changes to this package. You can leave this as the top line if you like, but you should add yourself so that future updates can be streamlined. (Look at other definitions in the repo to ensure you get the format for multiple owners correct.)